sfm_pc/management/commands/import_google_doc.py

```diff
@@ -672,242 +672,243 @@ def create_organization(self, org_data):
                                           org_data,
                                           organization)
 
-                # Create Compositions
-
-                try:
-                    parent_org_name = org_data[composition_positions['Parent']['value']]
-                except IndexError:
-                    parent_org_name = None
-
-                if parent_org_name:
-
-                    try:
-                        parent_confidence = self.get_confidence(org_data[composition_positions['Parent']['confidence']])
-                    except (IndexError, KeyError):
-                        self.log_error('Parent organization for {} does not have confidence'.format(organization.name))
-                        parent_confidence = None
+                if org_data['unit:relation_type'] == 'child':
```

Comment:
This diff is really ugly!!! The only change I made to these blocks is indentation: I use the value of `unit:relation_type` as a switch to create a composition or a membership.